bots/mlbridge/src/main/java/module-info.java

```diff
@@ -32,6 +32,8 @@
     requires org.openjdk.skara.webrev;
     requires org.openjdk.skara.network;
     requires org.openjdk.skara.version;
+    requires org.openjdk.skara.jbs;
```

Comment:
Do we really require jbs now?

Reply:
I tried to remove it but got an error while running the test cases. I tried to add jbs as a test gradle dependency but it still complained during compiling code before running the tests.
So I guess, it is needed along with the common module. Hence I did not make any changes.

Here is the error message if removed.
`> Task :bots:mlbridge:compileJava FAILED
error: module not found: org.openjdk.skara.jbs
1 error`